### PR TITLE
add default for function output

### PR DESCRIPTION
**Pull request by estute**
## Configuration Pull Request

Make sure that the following steps are done before merging
- [ ] @devops team member has commented with :+1:
- [ ] are you adding any new default values that need to be overriden when this goes live?  
  - [ ] Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
  - [ ] Add an entry to the CHANGELOG.

### CODE REVIEW

**Comment by estute** [comment]
@benpatterson @jibsheet @jzoldak

**Review by jzoldak — APPROVED** [review]
👍

**Comment by benpatterson** [comment]
👍

**Review by jibsheet — APPROVED** [review]

File: playbooks/roles/jenkins_worker/tasks/test_android_worker.yml [hunk 1]
```diff
@@ -28,7 +28,11 @@
 - name: Verify build tools installed
   stat: path={{ android_home }}/{{ item }}
   register: android_package_stats
-  with_items: "{{ android_test_paths.function_output }}"
+  # When this task is skipped, ansible still tries to resolve the variable
+  # returned from zip_to_list, but cannot. Add a default value to avoid these
+  # failures when other tasks are run, but not mask true failures when this
+  # task is run.
+  with_items: "{{ android_test_paths.function_output | default(['non-existent-package']) }}"
```
Comment (fredsmith) [inline playbooks/roles/jenkins_worker/tasks/test_android_worker.yml:35]: 1) why not put the default in roles/jenkins_worker/defaults/main.yml?

2) does {{ android_home }}/non-existent-package exist?  I suppose this task would register a null value if it doesn't, which is probably the desired outcome.

**Pull request merged**